### PR TITLE
access version from pacakge in gitcount

### PR DESCRIPTION
**Pull request by jGaboardi**
Addressing #653

### CODE REVIEW

**Comment by martinfleis** [comment]
One issue with this is that you now require whole pysal stack installed in the environment running gitcount, to import individual packages. I am not sure if this is used somewhere on CI or locally, so it may not actually be much of an issue.

**Comment by jGaboardi** [comment]
> One issue with this is that you now require whole pysal stack installed in the environment running gitcount, to import individual packages. I am not sure if this is used somewhere on CI or locally, so it may not actually be much of an issue.

Hmmmm. Point taken... I think I found a more elegant solution. Will push up now.

File: tools/gitcount.ipynb [hunk 1]
```diff
@@ -146,15 +146,28 @@
...
+     "end_time": "2021-09-19T21:40:58.557330Z",
+     "start_time": "2021-09-19T21:40:58.550183Z"
     }
    },
    "outputs": [],
    "source": [
     "# get __version__\n",
-    "import spaghetti\n",
-    "__version__ = spaghetti.__version__"
+    "run ../spaghetti/_version.py"
```
Comment (martinfleis) [inline tools/gitcount.ipynb:156]: How does this work? I get SyntaxError trying to run the cell and undefined `get_versions` below.

Comment (jGaboardi) [reply tools/gitcount.ipynb:156]: I somehow deleted the leading `%` it should be:
```python
%run ../spaghetti/_version.py
__version__ = get_versions()["version"]
```
This declares the `get_versions` function then isolates the versioneer version number like it does in [`__init__`](https://github.com/pysal/spaghetti/blob/075619c8bc30f9227efe1f61fdeb242ebbbbb956/spaghetti/__init__.py#L11). Pushing up the fix now.

Comment (martinfleis) [reply tools/gitcount.ipynb]: Smart, I didn't know about `%run` magic.

**Review by martinfleis — APPROVED** [review]

**Pull request merged**